### ISSUE TITLE
It just doesn't work

### ISSUE THREAD

**Issue by anlsergio** [delphiki/AirStatus#21]
I did everything as the instructions says:
- Tried running with `python3`
- Tried `python`
- Tried `sudo python....`
- Installed dependencies using `pip install -r requirements.txt`, `sudo pip`, `pip3`, `sudo pip3`
- Installed directly from the AUR repository https://aur.archlinux.org/packages/airstatus-git

The result is always the same:
- If I run it from the terminal, everything seems to work, no logs, no errors, just hanging there, but no output is printed out whatsoever (no logs nor output file is generated).
- If I run it as a service it spins up, all green and great, but no file output in `/tmp/airstatus.out`

I tried rebooting, tried keeping the lid of the case open while connected, but nothing, just nothing. I'd be happy to at least see an error to know what actually is going wrong, to find out at least where to start looking.

My system is Manjaro Gnome:

![image](https://github.com/delphiki/AirStatus/assets/54952031/dc4b782b-b2fd-423d-b206-732c98d0a909)

I don't see any recent activity around here, it's been more than a year since this repository was updated so I'm not sure if it's not working anymore or if it's just me missing something...

**Comment by delphiki** [comment]
Hello @anlsergio,

The project should still be functional (I'm using it right now).
Did you check the ControllerMode in you bluetooth config file (`/etc/bluetooth/main.conf`)?
It should be "dual":
```
ControllerMode = dual
```

**Comment by anlsergio** [comment]
> Hello @anlsergio,
> 
> The project should still be functional (I'm using it right now). Did you check the ControllerMode in you bluetooth config file (`/etc/bluetooth/main.conf`)? It should be "dual":
> 
> ```
> ControllerMode = dual
> ```

Yep, mine was set as `dual` by default (implicitly) but I tried by explicitly setting it as well, and the result is still the same. Sorry for not putting that in my original commentary, since it's relevant information too.

**Comment by anlsergio** [comment]
Bluetooth restart won't help either, because I tried many combinations of even restarting the computer altogether after the modifications made

**Comment by anlsergio** [comment]
So I managed to make it work!

![image](https://github.com/delphiki/AirStatus/assets/54952031/55b876b3-816c-4b49-ba42-d4801a4ec9e6)


It turns out that my sh**ty Bluetooth adapter is probably too weak which makes the Airpods signal poor enough to be excluded from the script filter:

By adding some print logs I've been able to see what was going wrong in my specific case so I changed the value of `MIN_RSSI` to `-100` so that my Airpods are under the threshold.

![image](https://github.com/delphiki/AirStatus/assets/54952031/6592efd9-01a4-460b-ac0e-e320cffcc324)

Perhaps it would be good to have this kind of information available from the README.md documentation or even give the possibility of the user to easily tweak it from a config file because I believe this might be the case of many people giving it a try and not being able to make it work.
